bin/interpreter.sh

```diff
@@ -128,6 +131,8 @@ if [[ "${INTERPRETER_ID}" == "spark" ]]; then
   fi
 fi
 
+addJarInDir "${LOCAL_REPO_DIR}"
```

Comment:
Once user loads the libraries, Zeppelin downloads them to `ZEPPELIN_HOME/local-repo` first, and copy only files to `ZEPPELIN_HOME/local-repo/{interpreterId}`. So every file under this directory will be file, not directory. Maybe naming is confusing. Will it help to change name of this variable to `${LOCAL_REPO_INTERPRETER_DIR}`?